planning/behavior_velocity_planner/occlusion-spot-design.md

```diff
@@ -208,51 +228,22 @@ note right
   convert from occupancy grid to image to use opencv functions.
 end note
 }
-partition offset_calculation {
-:interpolate ego path;
-note right
-  using spline interpolation and interpolate (x,y,z,v)
-end note
-:get closest index from ego position in interpolated path;
-:calculate offset from path start to ego;
-}
-:convert interpolated path to `path_lanelet`;
-note right
-  `path_lanelet` is lanelet which is created by ego path
-    and mainly used for arc coordinate conversion.
-end note
 partition generate_possible_collision {
+:calculate offset from path start to ego;
 :generate possible collision from occlusion spot;
 note right
   - occlusion spot candidate is N by N size unknown cells.
   - consider occlusion which is nearer than `lateral_distance_threshold`.
 end note
 :calculate collision path point and intersection point;
-note right
-  - occlusion spot is calculated by longitudinally closest point of unknown cells.
-  - intersection point is where ego front bumper and darting object will crash.
-  - collision path point is calculated by arc coordinate consider ego vehicle's geometry.
-end note
 }
-:extract target road type start/end distance by arc length;
-:calculate original velocity and height for the possible collision;
-note right
-calculated (x,y,z,v) at the path using linear interpolation between interpolated path points.
-end note
-partition calculate_safe_velocity {
+partition handle_possible_collision{
```

Comment:
Space is required before `{`

![image](https://user-images.githubusercontent.com/59680180/152281761-8a144c04-7de4-4b59-b88b-f4396aacf3ed.png)

Reply:
```suggestion
partition handle_possible_collision {
```